### DISCUSSION TITLE
Uninstall Windows

### DISCUSSION THREAD

**Discussion by meanin2** [semperai/amica#113] [General]
I did not create a new folder when i installed Amica on windows. 
I want to uninstall it, what specifically should i delete in order to uninstall it manually?

**Comment by yume-arasaki** [comment]
It depends how you installed Amica on windows, if by git clone then you should delete these files.

![image](https://github.com/user-attachments/assets/a8f1dda0-ac68-4b97-9e97-55e14c241ad7)

![image](https://github.com/user-attachments/assets/e4baeaae-0b13-49a5-978a-6ea1e960fcdf)

![image](https://github.com/user-attachments/assets/9e3115cd-5e0e-4bf2-be2d-47e401682030)
